kvin.js

```diff
@@ -45,18 +45,47 @@
 "use strict";
 
 {/* This prologue allows a CJS2 module's exports to be loaded with eval(readFileSync(filename)) */
-let originalModuleDeclare;
-if (typeof module === 'undefined' || typeof module.declare === 'undefined') {
-  originalModuleDeclare = (typeof module === 'object') ? module.declare : null
-  if (typeof module !== 'object') {
-    var module = { exports: {} }  // eslint-disable-line
-  }
-  module.declare = function moduleUnWrapper (deps, factory) {
-    factory(null, module.exports, module)
-    return module.exports
-  }
-}
-
+  var module;
```

Comment:
Won't this `var module` mask any global `module` variable (which is what we want to look at in the typeofs below)?